docs/source/10min_dask_cudf.ipynb

```diff
@@ -0,0 +1,2565 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 58,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "('a', 'b', 'e')"
+      ]
+     },
+     "execution_count": 58,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "gdf.grade.cat.categories"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Accessing the underlying code values of each categorical observation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 59,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0    0\n",
+      "1    1\n",
+      "2    1\n",
+      "3    0\n",
+      "4    0\n",
+      "5    2\n",
+      "dtype: int8\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(gdf.grade.cat.codes)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 60,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0    0\n",
+      "1    1\n",
+      "2    1\n",
+      "0    0\n",
+      "1    0\n",
+      "2    2\n",
+      "dtype: int8\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(dgdf.grade.cat.codes.compute())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plotting\n",
+    "------------\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Converting Data Representation\n",
+    "--------------------------------"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pandas"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Converting a cuDF and dask_cuDF `DataFrame` to a pandas `DataFrame`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 61,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "   a   b  c  agg_col1  agg_col2\n",
+      "0  0  19  0         1         1\n",
+      "1  1  18  1         0         0\n",
+      "2  2  17  2         1         0\n",
+      "3  3  16  3         0         1\n",
+      "4  4  15  4         1         0\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(df.head().to_pandas())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 62,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "   a   b  c  agg_col1  agg_col2\n",
+      "0  0  19  0         1         1\n",
+      "1  1  18  1         0         0\n",
+      "2  2  17  2         1         0\n",
+      "3  3  16  3         0         1\n",
+      "4  4  15  4         1         0\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(ddf.compute().head().to_pandas())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Numpy"
```

Comment:
Should add `df.to_matrix()` in this section as well.

Reply:
Is there a `to_matrix` method for dataframes to go to numpy? I may be missing it but I don't think there is one. Do you mean `to_gpu_matrix` to grab the devicearray?

Reply:
Addressed all comments except this one. Can include `as_gpu_matrix`, or something else. Defer to you @kkraus14

Reply:
Adding in `as_matrix` to convert to a standard numpy array.

Reply:
There is an `as_matrix()` method which is what I was referring to but typoed.